packages/auth-providers/azureActiveDirectory/web/package.json

```diff
@@ -30,11 +30,10 @@
     "@azure/msal-browser": "2.32.2",
     "@babel/cli": "7.20.7",
     "@babel/core": "7.20.12",
-    "@testing-library/react-hooks": "8.0.1",
```

Comment:
I think we need to add `"@testing-library/react"` here and in similar cases. Let me take a look at the tests in these packages.